src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaModelUtils.java

```diff
@@ -128,10 +128,10 @@ public static FlowPath buildPath(
     }
 
     /**
-     * Builds 2 PathSegment objects.
+     * Builds N PathSegment objects.
      */
-    public static List<PathSegment> buildSegments(PathId pathId, Switch switch1, Switch switch2, Switch switch3) {
-        return buildSegments(pathId, Lists.newArrayList(switch1, switch3, switch2));
+    public static List<PathSegment> buildSegments(PathId pathId, Switch... switches) {
+        return buildSegments(pathId, Lists.newArrayList(switches));
```

Comment:
> May be all the callers are initially expected the correct behaviour ?
not all of them. Okay. let's keep it as it. We can change in later, because PR is already very big